docs/user-guide/sources/kafka.md

```diff
@@ -57,3 +57,39 @@ spec:
               # Set this to false if using a non-Kafka SASL proxy
               handshake: true 
 ```
+
+## FAQ
+### How to start the Kafka Source from a specific offset based on datetime?
+In order to start the Kafka Source from a specific offset based on datetime, we need to reset the offset before we start the pipeline.
+
+For example, we have a topic `quickstart-events` with 3 partitions and a consumer group `console-consumer-94457`. This example uses [Kafka 3.6.1](https://downloads.apache.org/kafka/3.6.1/RELEASE_NOTES.html).
+```shell
+➜  kafka_2.13-3.6.1 bin/kafka-topics.sh --bootstrap-server localhost:9092 --describe --topic quickstart-events            
+Topic: quickstart-events	TopicId: WqIN6j7hTQqGZUQWdF7AdA	PartitionCount: 3	ReplicationFactor: 1	Configs: 
+	Topic: quickstart-events	Partition: 0	Leader: 0	Replicas: 0	Isr: 0
+	Topic: quickstart-events	Partition: 1	Leader: 0	Replicas: 0	Isr: 0
+	Topic: quickstart-events	Partition: 2	Leader: 0	Replicas: 0	Isr: 0
+```
+```shell
+➜  kafka_2.13-3.6.1 bin/kafka-consumer-groups.sh --bootstrap-server localhost:9092 --list --all-groups                                                                                                     
+console-consumer-94457
+```
+We have already consumed all the available messages in the topic `quickstart-events`, but we want to go back to some datetime and re-consume the data from that datetime.
+```shell
+➜  kafka_2.13-3.6.1 bin/kafka-consumer-groups.sh --bootstrap-server localhost:9092 --describe --group console-consumer-94457                          
+
+GROUP                  TOPIC             PARTITION  CURRENT-OFFSET  LOG-END-OFFSET  LAG             CONSUMER-ID     HOST            CLIENT-ID
+console-consumer-94457 quickstart-events 0          56              56              0               -               -               -
+console-consumer-94457 quickstart-events 1          38              38              0               -               -               -
+console-consumer-94457 quickstart-events 2          4               4               0               -               -               -
+```
+To achieve that, before the pipeline start, we need to first stop the consumers in the consumer group `console-consumer-94457` because offsets can only be reset if the group `console-consumer-94457` is inactive. Then, reset the offsets using the desired date and time. The example command below uses UTC time.
+```shell
+➜  kafka_2.13-3.6.1 bin/kafka-consumer-groups.sh --bootstrap-server localhost:9092 --execute --reset-offsets --group console-consumer-94457 --topic quickstart-events --to-datetime 2024-01-19T19:26:00.000
```

Comment:
1. Please verify this is working from anywhere (not just kafka server).
2. https://docs.confluent.io/kafka/operations-tools/use-kafka-tools-ccloud.html#use-kafka-tool-ccloud
3. https://docs.confluent.io/kafka/operations-tools/manage-consumer-groups.html